guidelines/sc/20/visual-presentation.html

```diff
@@ -13,5 +13,6 @@ <h4>Visual Presentation</h4>
      <li>Line spacing (leading) is at least space-and-a-half within paragraphs, and paragraph spacing is at least 1.5 times larger than the line spacing.</li>   
      <li>Text can be resized without assistive technology up to 200 percent in a way that does not require the user to scroll horizontally to read a line of text <a>on a full-screen window</a>.</li>   
   </ul>
+  <p class="note">The 'mechanism' is usually the browser or assitive technology manipulating the content on behalf of the user. It is not a requirement for the author to provide the mechanism. Some writing systems have additional or different requirements, or do not customarily use features such as paragraph spacing in which case users might not use some of these mechanisms.</p>
```

Comment:
Suggestion to change "browser" to "user agent", I mention this only because browsers are not the only user agents that do this. Other user agents that do this may include reading systems, word processing tools (outside of the browser), etc.